cli/testing/fake-editor.rs

```diff
@@ -50,6 +50,12 @@ fn main() {
                 let dest_path = edit_script_path.parent().unwrap().join(dest);
                 fs::copy(&args.file, dest_path).unwrap();
             }
+            ["dumppath", dest] => {
+                let dest_path = edit_script_path.parent().unwrap().join(dest);
+                fs::write(&dest_path, args.file.to_str().unwrap()).unwrap_or_else(|_| {
+                    panic!("Failed to write file {}", args.file.to_str().unwrap())
```

Comment:
nit: include the source error (currently ignored with `_`) in the message to help anyone troubleshooting this in the future for some reason?

Reply:
Done - and also fixed the incorrect file here (it should be dest_path).